### PR TITLE
Improve logging on Android

### PR DESCRIPTION
**Pull request by at-ninja**
Additionally, add tests to verify logging to console.

(Re-try at https://github.com/google/bigwheels/pull/84)

### CODE REVIEW

**Review by vettoreldaniele — COMMENTED** [review]
Thanks a lot for adding tests! Just a few comments

File: android/gradlew
[the remark this answers is not in the document]
Comment (at-ninja) [reply android/gradlew:1]: It's a `chmod a+x` on the shell script, to allow for ease of use on the command line.

I can revert it, but I find it makes it easier to use the command line to build the android app.

File: src/ppx/log.cpp [hunk 1]
```diff
@@ -151,24 +153,70 @@ void Log::DestroyObjects()
...
             OutputDebugStringA(msg);
         }
         else {
-            std::cout << msg;
+            *mConsoleStream << levelString << msg;
         }
+#elif defined(PPX_ANDROID)
+        android_LogPriority prio;
+        switch (level) {
+            case LOG_LEVEL_INFO:
```
[the remark this answers is not in the document]
Comment (at-ninja) [reply src/ppx/log.cpp]: Done.

File: android/gradlew
[the remark this answers is not in the document]
Comment (vettoreldaniele) [reply android/gradlew:1]: Thanks for the explanation, no need to revert, sounds useful!

**Review by vettoreldaniele — APPROVED** [review]

**Pull request merged**